routes/index.js

```diff
@@ -1,18 +1,16 @@
-var express = require('express');
-var router = express.Router();
+import express from 'express';
+const router = express.Router();
 
-var author = require('routes/author');
-var mail = require('routes/mail');
-var api = require('routes/api');
+import author from 'routes/author';
+import mail from 'routes/mail';
+import api from 'routes/api';
 
-var log = require('common/log');
-
-router.get('/', function(req, res) {
-  var isLogin = req.session.passport.user;
+router.get('/', (req, res)=> {
+  const isLogin = req.session.passport.user;
```

Comment:
__[jshint]__
*  'const' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).